### PR TITLE
v1.9: Update cilium base images

### PR DESCRIPTION
**Pull request by joestringer**
Bump the cilium-runtime base images to pick up the latest set of underlying packages.

Related: #17264

### CODE REVIEW

**Review by michi-covalent — APPROVED** [review]

**Review by twpayne — APPROVED** [review]

**Comment by twpayne** [comment]
test-backport-1.9

**Comment by joestringer** [comment]
Jenkins CI completely failed, but it should not be anything to do with the changes here since the smoke tests pass. This is just a rebuild of existing images which should have already passed CI, but bumping the underlying package versions.

**Comment by joestringer** [comment]
test-backport-1.9

File: jenkinsfiles/ginkgo-gke.Jenkinsfile [hunk 1]
```diff
@@ -30,7 +30,7 @@ pipeline {
             )}"""
         BASE_IMAGE="""${sh(
                 returnStdout: true,
-                script: 'if [ "${run_with_race_detection}" = "" ]; then echo -n "scratch"; else echo -n "quay.io/cilium/cilium-runtime:2021-08-21-v1.9@sha256:5939ee503abebba75167db547f8ad9bf8f99c6ead7eb6c031b4170588a0ae0b0"; fi'
+                script: 'if [ "${run_with_race_detection}" = "" ]; then echo -n "scratch"; else echo -n "quay.io/cilium/cilium-runtime:2021-08-30-v1.9@sha256:10a3ce8226135882a53e91a803819277a35260bc3419f75cb20e5e359a3d4662
```
Comment (nbusseneau) [inline jenkinsfiles/ginkgo-gke.Jenkinsfile]: Missing `"; fi'` at EOL :D
Likewise for other Jenkinsfiles.

**Review by nbusseneau — CHANGES_REQUESTED** [review]

File: jenkinsfiles/ginkgo-gke.Jenkinsfile (hunk 1 above)
Comment (joestringer) [reply jenkinsfiles/ginkgo-gke.Jenkinsfile]: Good spotting, thanks. For v1.8/v1.9 I had to use `sed` to inject these, I've updated my expression to:

```
sed -i 's/\(cilium-runtime\):[-:.@0-9a-zA-Z]\+/\1:2021-08-30-v1.9@sha256:10a3ce8226135882a53e91a803819277a35260bc3419f75cb20e5e359a3d4662/g' $(git grep -l cilium-runtime)
```

**Comment by joestringer** [comment]
test-backport-1.9

**Review by nbusseneau — APPROVED** [review]

**Comment by joestringer** [comment]
[test-gke](https://jenkins.cilium.io/job/Cilium-PR-K8s-GKE/6318/): Known image pull issue (#17282)
[test-1.12-net-next](https://jenkins.cilium.io/job/Cilium-PR-K8s-1.12-net-next/1804/execution/node/131/log/): Timed out during cluster provisioning
[test-1.15-4.9](https://jenkins.cilium.io/job/Cilium-PR-K8s-1.15-kernel-4.9/372/): A pod gets deleted, but the endpoint remains and this causes test failures: #17283. Must be already a bug in the tree, there's no obvious reason why this PR could impact that since this PR is only rebuilding the images with the exact same code but different underlying ciliium-runtime image.
[test-upstream-k8s](https://jenkins.cilium.io/job/Cilium-PR-K8s-Upstream/3302/execution/node/52/log/): VM pull timed out

Ignoring failures to merge & unblock security releases.

**Pull request merged**